### DISCUSSION TITLE
Keyboard shortcuts conflict

### DISCUSSION THREAD

**Discussion by xuanzhi33** [community/community#130354] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

In VS Code, the default keyboard shortcut for opening the Copilot panel is <kbd>Ctrl</kbd>+<kbd>Enter</kbd>, which conflicts with the frequently used shortcut of `Create new line` (<kbd>Ctrl</kbd>+<kbd>Enter</kbd>). I suggest making modifications to it.

**Comment by alexisseurin** [comment]
You can change keybindings directly on VSCode.

![image](https://github.com/community/community/assets/44972852/04b29f66-4474-4094-970e-9ad6835a7a76)
